pepper-apis/docs/specification/src/endpoints/user.studies.status.yml

```diff
@@ -21,5 +21,11 @@ get:
   responses:
     200:
       $ref: '../pepper.yml#/components/responses/ParticipantStatusResponse'
+    404:
```

Comment:
API is already returning a 404, so while I'm in here, just updating the docs to reflect reality.